README.md

```diff
@@ -27,35 +29,74 @@ expirationd.start(job_name, space.id, is_expired, {
 })
 ```
 
+Сustomized version:
+```lua
+expirationd.start(job_name, space.id, is_expired, {
+    -- default is primary key
+    index = 'exp',
+    -- one transaction per batch
+    -- default is false
+    atomic_iteration = true,
+    -- delete data that was added a year ago
+    -- default return nil
+    start_key = function( task )
+        return clock.time() - (365*24*60*60)
+    end,
+    -- delete it from the oldest to the newest
+    -- default is ALL
+    iterator_type = 'GE',
+    -- stop full_scan if delete a lot
+    -- default return true
+    process_while = function( task )
+        if task.args.max_expired_tuples >= task.expired_tuples_count then
+            task.expired_tuples_count = 0
+            return false
+        end
+        return true
+    end,
+    -- this function is default if no other is specified
+    iterate_with = function( task )
+        return task.expire_index:pairs({ task.start_key() }, { iterator = task.iterator })
+            :take_while( function( tuple )
+                return task:process_while()
+            end )
+    end,
+    args = {
+        max_expired_tuples = 1000
+    }
+})
+```
+
 ## Expirationd API
 
 ### `expirationd.start (name, space_id, is_tuple_expired, options)`
 
-Run a named task
+Run a scheduled task to check and process (expire) tuples in a given space.
 
 * `name` - task name
 * `space_id` - space to look in for expired tuples
 * `is_tuple_expired` - a function, must accept tuple and return true/false
   (is tuple expired or not), receives `(args, tuple)` as arguments
 opt
```

Comment:
A strange leftover?

Reply:
I didn't quite understand what it meant. Write options instead opt?

Reply:
This option description needs rewriting

Reply:
deleted unnecessary words